### ISSUE TITLE
Support for SQL Server

### ISSUE THREAD

**Issue by pmairoldi** [megaease/easeprobe#589]
I've been looking into it for my monitoring needs and I'm really impressed so far! Great work on the tool!  

One thing I would need if I wanted to montior everything is SQL Server support. I'd be happy to take a stab at implementing it if that is something that you would be open to supprt. 

After looking a bit into the code I think it wouldn't be too difficult. We would need to add a new client probe for SQL Server, implement all the required methods and then expose it to the client configuration code. 

If you are open to it ill start looking into it more deeply!

**Comment by samanhappy** [comment]
Absolutely, contributions are welcome, and we're looking forward to your implementation!

**Comment by pmairoldi** [comment]
Turns out I will it be needing this after all. Closing
